cmd/start/config.go

```diff
@@ -112,3 +121,31 @@ func MustNewConfig(v *viper.Viper) *Config {
 
 	return config
 }
+
+func sliceTypeEnvDecodeHookFunc[T any](from, to reflect.Value) (any, error) {
```

Comment:
I'd prefer to move the hooks to a separate package so that all cli commands can use them

Reply:
I moved them to a new hooks package.
Also, I removed the specific system users hook and use the generic map hook instead. Probably, the system users hook came in when I updated the branch and I overlooked it.